mungegithub/mungers/e2e/e2e.go

```diff
@@ -264,7 +266,10 @@ func (e *RealE2ETester) LatestRunOfJob(jobName string) (int, error) {
 
 // LoadNonBlockingStatus gets the build stability status for all the NonBlockingJobNames.
 func (e *RealE2ETester) LoadNonBlockingStatus() {
-	for _, job := range e.NonBlockingJobNames {
+	e.Opts.Lock()
+	jobs := *e.NonBlockingJobNames
```

Comment:
Mungers do not modify option values except in tests and the options module assigns a new slice when it updates a `[]string` option (it doesn't change the values in the underlying array) so there shouldn't be a race here.